python/src/flow_graph.cpp

```diff
@@ -112,6 +112,12 @@ add_flow_graph_bindings(py::module& m)
         .def_property_readonly("dfs_indices",
                                &fs::py_flow_graph_impl::dfs_indices,
                                "Deprecated alias of :attr:`FlowGraphImpl.nodes_indices_bottomup`.")
+        .def_property_readonly("bfs_indices",
+                               &fs::py_flow_graph_impl::bfs_indices,
+                               "Deprecated alias of :attr:`FlowGraphImpl.nodes_indices_bottomup`.")
```

Comment:
The description is not 100% accurate (anyway we'll need to reconsider the API later when implementing lazy graph traversal, e.g., a `nodes_indices()` method with a `FlowGraphTraversalDir` argument may be more appropriate).

Reply:
it's a bad copy/paster